framework/src/bundle/CoreBundleContext.h

```diff
@@ -187,6 +199,11 @@ namespace cppmicroservices
 
         void Uninit1();
 
+        WriteLock SetFrameworkStopped();
+
+        WriteLock SetFrameworkStarted();
+
+        std::unique_ptr<BlockFrameworkShutdown> GetFrameworkStopped() const;
```

Comment:
`GetFrameworkStopped` is not a descriptive name for what this method is meant to do. This method blocks the `stopped` flag from being modified until `BlockFrameworkShutdown` is destroyed.

Reply:
Changed the names... I think they are better, but probably not quite right yet